bundles/org.openhab.binding.boschshc/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -259,6 +272,7 @@
 		<label>Climate Control</label>
 		<description>This is a virtual device which is automatically created for all rooms that have
 			thermostats in it.</description>
+		<semantic-equipment-tag>Thermostat</semantic-equipment-tag>
```

Comment:
Does `Thermostat` refer only to the device that controls the temperature directly at the radiator or also to control devices for thermostats? If both are meant, `Thermostat` is fine. Otherwise we could consider `ControlDevice`. Note that this is a virtual device that is bound to rooms, not a physical device.

Reply:
Both are meant.